test/integration/helpers/serverspec/shared_spec.rb

```diff
@@ -35,12 +51,34 @@
       it 'xpack should be activated' do
         expect(curl_json("#{es_api_url}/_license", username=username, password=password)['license']['status']).to eq('active')
       end
-      features = curl_json("#{es_api_url}/_xpack", username=username, password=password)
-      curl_json("#{es_api_url}/_xpack", username=username, password=password)['features'].each do |feature,values|
-        enabled = vars['es_xpack_features'].include? feature
-		status = if enabled then 'enabled' else 'disabled' end
-		it "the xpack feature '#{feature}' to be #{status}" do
-          expect(values['enabled'] = enabled)
+      if vars.key?('es_xpack_features')
+        curl_json("#{es_api_url}/_xpack", username=username, password=password)['features'].each do |feature,values|
+          enabled = vars['es_xpack_features'].include? feature
+          status = if enabled then 'enabled' else 'disabled' end
+          it "the xpack feature '#{feature}' to be #{status}" do
+            expect(values['enabled'] = enabled)
+          end
+        end
+      else
+        features.each do |feature, status|
```

Comment:
Whats the idea of wanting to test this? If this were to start failing it would be because of a changed default in an Elasticsearch version. If that were to happen we would just be updating the defaults in this test file right?

Reply:
The idea was to still have a test when `es_xpack_features` variable is empty but I can remove it if you think it's not relevant.

Reply:
strange that you saw a change in this PR because this is a change from https://github.com/elastic/ansible-elasticsearch/pull/560/files which is already merged